moe._999eagle.OkunaDesktop.json

```diff
@@ -0,0 +1,82 @@
+{
+	"app-id": "moe._999eagle.OkunaDesktop",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "18.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "okuna-desktop.sh",
+	"finish-args": [
+		/* X11 */
+		"--share=ipc", "--socket=x11",
+		/* Network */
+		"--share=network",
+		/* OpenGL */
+		"--device=dri",
+		/* PulseAudio */
+		"--socket=pulseaudio"
+	],
+	"modules": [
+		{
+			"name": "okuna-desktop",
+			"buildsystem": "simple",
+			"build-commands": [
+				"./build.sh"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://gitlab.com/999eagle/okuna-desktop/-/jobs/343804367/artifacts/download",
+					"dest": "okuna-desktop",
+					"dest-filename": "artifacts.zip",
+					"sha256": "f4c6bedf0f4156eb44175b941fc87991dcc9abdbdbfc5aec9e42cdeae2d487b0"
+				},
+				{
+					"type": "file",
+					"path": "build.sh"
+				},
+				{
+					"type": "file",
+					"path": "okuna-desktop.sh"
+				},
+				{
+					"type": "file",
+					"path": "moe._999eagle.OkunaDesktop.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Okay, will change that.